src/dotnet-core-uninstall/Shared/VSVersioning/VisualStudioSafeVersionsExtractor.cs

```diff
@@ -14,9 +16,12 @@ internal static class VisualStudioSafeVersionsExtractor
         // Divisions within version bands. Not inclusive, groupings: [2.1.000, 2.1.600) [2.1.600, 2.2.00) [2.0.00, 2.2.200) [2.2.200, 2.3.00)
         private static readonly SemanticVersion[] SpecialCaseDivisions = { new SemanticVersion(2, 1, 600), new SemanticVersion(2, 2, 200) };
 
+        // The tool should not be used to uninstall any more recent versions of the sdk
+        private static readonly SemanticVersion UpperLimit = new SemanticVersion(3, 0, 0);
```

Comment:
@KathleenDollard should we also put this somewhere in help to also let the user know why this tool cannot remove 3.0 (post 3.0 it is handled properly by VS)